backend/src/main/java/com/google/backend/servlets/AlertVisualizationServlet.java

```diff
@@ -46,4 +54,39 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       throw new ServletException(e);
     }
   }
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) 
+      throws ServletException, IOException {
+    String[] data = processRequestBody(request);
+    Long id = Long.parseLong(data[0]);
+    String priority = data[1];
```

Comment:
i thought it was "statusToChangeTo" and not priority

Reply:
typo :'(